include/dlr_tvm.h

```diff
@@ -27,6 +27,9 @@ class DLR_DLL TVMModel : public DLRModel {
   std::vector<std::string> output_types_;
   std::vector<std::string> weight_names_;
   void SetupTVMModule(std::vector<std::string> model_path);
+  void SetupTVMModule(const ModelPath& paths);
+  void SetupTVMModule(const std::string& json_str, const std::string& param_str,
```

Comment:
Could you make `SetupTVMModule` take a `dmlc::Stream` instead of string for param_str?

That way it can handle this change in the custom allocator PR:
https://github.com/neo-ai/neo-ai-dlr/pull/269/files#diff-0c87eebc446be9132c7f9bae311fa7b6b339bac8d650e23e7569b9561f8c7fdaR79-R80

When you call SetupTVMModule, for now you can use
`SetupTVMModule(json_str, dmcl::MemoryStringStream(&param_str), paths);`

Reply:
done